CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.30.6](https://github.com/alexa/ask-cli/compare/v2.30.4...v2.30.6) (2023-12-09)
+
+
```

Comment:
do you want to wait for the [new PR](https://github.com/alexa/ask-cli/pull/500) to be merged or want to follow up with another one to update the changelog?

Reply:
Lets merge your PR then update this one

Reply:
extra stuff 

```
### Features

* add ASK_FORCE_ENABLE to allow skipping domain validation during enablement ([487f8db](https://github.com/alexa/ask-cli/commit/487f8dbcdc9429d04b6fd475aa4a2a8430831f1f))

```

Reply:
updated the commit